client/spec/crousel.test.js

```diff
@@ -0,0 +1,28 @@
+// import Enzyme from 'enzyme';
```

Comment:
Small note - the file name has a typo

Reply:
Nice catch aarushi, I'll edit that right away.